core/src/main/java/bisq/core/api/CoreOffersService.java

```diff
@@ -87,15 +88,16 @@ public CoreOffersService(CreateOfferService createOfferService,
     }
 
     // Create and place new offer.
-    Offer createOffer(String currencyCode,
-                      String directionAsString,
-                      String priceAsString,
-                      boolean useMarketBasedPrice,
-                      double marketPriceMargin,
-                      long amountAsLong,
-                      long minAmountAsLong,
-                      double buyerSecurityDeposit,
-                      String paymentAccountId) {
+    Offer createAndPlaceOffer(String currencyCode,
```

Comment:
I think it would be more clear to not return the offer, as it is passed via the `Consumer<Offer> resultHandler`.

Reply:
Clever.   Fix is in commit 30e6dae.